posts/2019-01-22-the-5-raisons-detre-of-testing.md

```diff
@@ -0,0 +1,219 @@
+---
+title: The 5 Raisons d'Être of Testing
+---
+
+<style>
+.story {
+  background-color: lightblue;
+  padding: 20px;
+  border-radius: 10px;
+}
+
+</style>
+
+# The 5 Raisons d'Être of Testing
+
+I describe five of the reasons why I think tests are important:
+
+* Ensure that the code you've written works.
+* Ensure that the code you've written works tomorrow.
+* Ensure that the code you've written works elsewhere.
+* Spot interface issues early on.
+* Show how your feature is supposed to work, and how it should be used.
```

Comment:
some types of tests can be useful to guide your reasoning indeed, not all of them though